### PR TITLE
DIV-6907: Making improvements and fixes in existing tests

### PR DESCRIPTION
**Pull request by jvictorcf**
# Description

Sign out functionality will be fixed as part of this ticket, so we're skipping its tests for now.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: test/end-to-end/pages/grounds-for-divorce/unreasonable-behaviour.js [hunk 1]
```diff
@@ -10,12 +10,7 @@ function enterUnreasonableBehaviourExample(language = 'en') {
   I.waitInUrl(pagePath);
   I.seeInCurrentUrl(pagePath);
   I.fillField('reasonForDivorceBehaviourDetails[]', mockSession.reasonForDivorceBehaviourDetails[0]);
-
-  if (language === 'en') {
-    I.navByClick(commonContent.continue);
-  } else {
-    I.navByClick(commonContent.continue);
-  }
```
Comment (jacmurphy) [inline test/end-to-end/pages/grounds-for-divorce/unreasonable-behaviour.js:18]: image

Im glad you cleaned this up!

File: test/end-to-end/paths/reasonsForDivorce.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ languages.forEach(language => {
     } else {
       await I.checkMyAnswersAndValidateSession(language);
     }
-
+
```
Comment (jacmurphy) [inline test/end-to-end/paths/reasonsForDivorce.js:56]: 👀

File: test/end-to-end/paths/reasonsForDivorce.js [hunk 3]
```diff
@@ -102,7 +101,7 @@ languages.forEach(language => {
     } else {
       await I.checkMyAnswers(language);
     }
-
+
```
Comment (jacmurphy) [inline test/end-to-end/paths/reasonsForDivorce.js:104]: Please remove

File: test/end-to-end/paths/reasonsForDivorce.js [hunk 4]
```diff
@@ -149,7 +147,7 @@ languages.forEach(language => {
...
```
Comment (jacmurphy) [inline test/end-to-end/paths/reasonsForDivorce.js:150]: Please remove

**Review by jacmurphy — APPROVED** [review]

**Review by yarpo — APPROVED** [review]

File: test/end-to-end/paths/reasonsForDivorce.js (hunk 3 above)
Comment (jvictorcf) [reply test/end-to-end/paths/reasonsForDivorce.js:104]: That's automatic styling guidelines (from Code).

**Pull request merged**